render.util.js

```diff
@@ -0,0 +1,45 @@
+export function renderBeanieBabies(beanieBaby) {
+    const beaniebabyEl = document.createElement('li');
+    beaniebabyEl.classList.add('card');
+
+    const img = document.createElement('img');
+    img.src = beanieBaby.image;
+    img.alt = beanieBaby.title;
```

Comment:
Love that you are thinking about accessibility and including the alt for the image!